packages/@aws-cdk/aws-appmesh/lib/route-spec.ts

```diff
@@ -68,6 +69,76 @@ export interface HttpRouteSpecOptions {
    * @default - None
    */
   readonly timeout?: HttpTimeout;
+
+  /**
+   * The retry policy
+   * @default - no retry policy
+   */
+  readonly retryPolicy?: HttpRetryPolicy;
+}
+
+/**
+ * HTTP retry policy
+ */
+export interface HttpRetryPolicy {
+  /**
+   * Specify HTTP events on which to retry
+   * @default - no retries for http events
```

Comment:
The fact that either this, or `tcpRetryEvents` is required, needs to be stated in the documentation of this property.